portfolio/src/main/java/com/google/sps/servlets/Logout.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+/**
+ * Logout the user from their account. 
+ */
+@WebServlet("/logout")
```

Comment:
When I log out, I get sent to a page with only the text:
```
Redirecting you to https://accounts.google.com/Logout?continue=https://appengine.google.com/_ah/logout%3Fcontinue%3Dhttps://google.com/url%253Fsa%253DD%2526q%253Dhttp://morleyd-step-2020.appspot.com/index.html%2526ust%253D1594492513570673%2526usg%253DAFQjCNFGHtNfW_7CGUeu506LDWMtrgVvMg&service=ah
```
I never get redirected to `index.html`, although a Chrome OS settings window does pop up (no idea what's happening there). Do you see the same behavior?

Reply:
Hmm this definitely worked for me on the local page, some people were complaining that the api when deployed will log you out of all your instances of your gmail. It seemed to work for me though when I tried it on both the local and the deployed server. I'll look into this.

Reply:
I still see the same behavior, but I've also seen it on the other portfolio pages. So don't worry if you can't figure it out, it may be a quirk with the API that isn't covered in the tutorial.